src/vma/main.cpp

```diff
@@ -241,6 +241,7 @@ static int free_libvma_resources()
 		//cosmetics - remove when adding iomux block
 		fprintf(g_stats_file, "======================================================\n");
 		fclose (g_stats_file);
+		g_stats_file = NULL;
```

Comment:
why `free_libvma_resources` is called twice

Reply:
Once from main_destroy() and from do_global_ctors() while vma_exception is caught.